### ISSUE TITLE
Code: Remove all redundant "normal table" code (filter, datatypes...) in files related to appointment booking system

### ISSUE THREAD

**Issue by Theophilix** [Theophilix/event-table-edit#53]
We have to remove for example pagination, dropdowns, filter, sorting...
```
models\appointmentform.php
models\appointments.php
site\views\appointments\*.php
site\views\appointmentform\*.php
```

**Comment by Theophilix** [comment]
This is linked to this bug: https://github.com/Theophilix/event-table-edit/issues/59

**Comment by Theophilix** [comment]
Fixed in 4.8.5 version,
